bin/ripple/ecdsa/__init__.py

```diff
@@ -0,0 +1,14 @@
+__all__ = ["curves", "der", "ecdsa", "ellipticcurve", "keys", "numbertheory",
+           "test_pyecdsa", "util", "six"]
+from .keys import SigningKey, VerifyingKey, BadSignatureError, BadDigestError
+from .curves import NIST192p, NIST224p, NIST256p, NIST384p, NIST521p, SECP256k1
+
+_hush_pyflakes = [SigningKey, VerifyingKey, BadSignatureError, BadDigestError,
+                  NIST192p, NIST224p, NIST256p, NIST384p, NIST521p, SECP256k1]
+del _hush_pyflakes
```

Comment:
What I mean is that it's _provable_ that removing lines 6 through 8 would have no discernable effect on the program.  Items don't know that they're being added to a list;  making a list has no side effects;  the list is immediately deleted.

I just want to make sure that this isn't some editing or copying error somewhere.